test/e2e/tools/eks-anywhere-test-tool/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/aws/eks-anywhere-test-tool
 
-go 1.16
+go 1.17
```

Comment:
Did you try 1.18?

Reply:
@chrisdoherty4, this is not the permanent upgrade to the latest version. The destination branch is a custom branch. I have created this branch to get eks-anywhere to be compatible with eks-curated-packages which currently uses go version 1.17.